Sources/Frisbee/Requestables/NetworkGetter.swift

```diff
@@ -19,8 +19,20 @@ public class NetworkGetter: Getable {
         makeRequest(url: url, onComplete: onComplete)
     }
 
+    public func get<Entity: Decodable>(url: URL, onComplete: @escaping (Result<Entity>) -> Void) {
+        makeRequest(url: url, onComplete: onComplete)
+    }
+
     public func get<Entity: Decodable, Query: Encodable>(url: String, query: Query,
                                                          onComplete: @escaping (Result<Entity>) -> Void) {
+        guard let url = URL(string: url) else {
+            return
```

Comment:
Is necessary to call `onComplete` with `fail` and `FrisbeeError.invalidUrl` error.